lib/breadcrumbs_on_rails/action_controller.rb

```diff
@@ -28,7 +29,15 @@ def add_breadcrumb(name, path = nil, options = {})
     end
 
     def breadcrumbs
-      @breadcrumbs ||= []
+      self.breadcrumbs_list.last
+    end
+
+    def next_breadcrumbs_list
```

Comment:
[MUST]
```suggestion
    def next_breadcrumbs
```

の間違いでしょうか？ `breadcrumbs_list` が配列の配列だと思うので、ここでは次の配列 (breadcrumns) に移るの意味だと思います。